src/app/globals.scss

```diff
@@ -34,7 +34,7 @@ main {
 }
 
 a {
-  color: inherit;
+  color: white;
```

Comment:
This could be $second-color. It is still white but it is easier to change if we want to change all white to a different color.

Reply:
Now changed